packages/flutter/lib/src/rendering/sliver_fill.dart

```diff
@@ -113,7 +113,17 @@ class RenderSliverFillRemaining extends RenderSliverSingleBoxAdapter {
   /// the remaining space in the viewport.
   RenderSliverFillRemaining({
     RenderBox child,
-  }) : super(child: child);
+    @required this.scrollBody,
```

Comment:
The name should indicate that this is a boolean, maybe "hasScrollBody or "isScrollable"?